model/versioning/Versioned.php

```diff
@@ -1556,16 +1647,7 @@ public static function set_reading_mode($mode) {
 	 * @return string
 	 */
 	public static function get_reading_mode() {
-		return Versioned::$reading_mode;
-	}
-
-	/**
-	 * Get the name of the 'live' stage.
-	 *
-	 * @return string
-	 */
-	public static function get_live_stage() {
```

Comment:
On second thoughts, we can delete it, so let's.

If we deprecate it, we should port it back to 3 branch and maybe raise the deprecation notice there.